graph/src/components/store.rs

```diff
@@ -603,6 +603,13 @@ pub trait Store: Send + Sync + 'static {
     /// Looks up an entity using the given store key.
     fn get(&self, key: EntityKey) -> Result<Option<Entity>, QueryExecutionError>;
 
+    /// Look up multiple entities. Returns a map of entities by type.
+    fn get_many(
```

Comment:
Instead of a new method in the `Store`, why not just use an `EntityQuery` with an `EntityFilter::In` as the filter?

Reply:
That'd be nice, but this can return from multiple entities types in one query, which `EntityQuery` can only do in a way makes sense if they implement the same interface.

Reply:
The storage layer actually has no idea about interfaces; all `EntityQuery` wants is that all the entity types mentioned have all the attributes that are used in the filter and the order. And since you are only filtering by `id`, that's true. Even when we query for entities that implement an interface, we get the entire concrete entity where some attributes might only appear on one of the entity types (justcan't filter/sort by those)

Reply:
Just to be clear: there's code in the relational schema that tracks which interfaces a table implements, but that information isn't used anywhere, and we should probably just delete it since it serves no real purpose.

Reply:
I mean that if I use an `EntityQuery` as we do for interfaces, I think I'll get wrong results because I can't express which entities are associated with which ids, if there is `Fred` the `Dog` and `Fred` the `Cat`, I can either have both or none, I can't have only one. Maybe I could use the `EntityQuery` to give me a superset of what I want and then filter, but that seemed hacky to me.

Reply:
You are right - `EntityQuery` isn't up to the job; I was hoping we could avoid adding another method to the `Store` trait, but you are right in that it is needed.